src/backend/distributed/sql/citus--10.1-1--10.2-1.sql

```diff
@@ -3,6 +3,14 @@
 -- bump version to 10.2-1
 
 DROP FUNCTION IF EXISTS pg_catalog.stop_metadata_sync_to_node(text, integer);
+GRANT ALL ON FUNCTION pg_catalog.worker_record_sequence_dependency(regclass,regclass,name) TO PUBLIC;
+
+-- the same shard cannot have placements on different nodes
+ALTER TABLE pg_catalog.pg_dist_placement ADD CONSTRAINT placement_shardid_groupid_unique_index UNIQUE (shardid, groupid);
```

Comment:
This unique index now makes our existing (non unique) index on `shardid` pretty useless. Can we drop that one and use this one in all places instead? This could also be done in a separate PR (especially if it's not as trivial as I think).

Reply:
Yeah, lets do it in a separate PR, this is already way too big :)

Reply:
Well, it actually doesn't make it useless right? We still do lookups with shardId on pg_dist_placement? Especially `BuildShardPlacementList` uses it and it is called during metadata loading. So, I'd be more comfortable to keep it

Reply:
I meant, you can do efficient lookups by shardId using the backing index for this constraint now.